src/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import AuthApiService from '../../services/auth-api-service';
+import UserContext from '../../context/UserContext';
+
+import './LoginForm.css'
+
+export default class LoginForm extends Component {
+  static defaultProps = {
+    onLoginSuccess: () => { }
+  }
+
+  static contextType = UserContext;
+
```

Comment:
i think we're missing a handler for submit here. I think spaced-repetition has a good base to start us off on. 
we can work on this together if you'd like.

Also, you created a FormUtil helper components and those would be useful here.

If you don't mind, let's try to pair program this thing: it should be quick

Reply:
hey, Nick I'm just waiting for the backend to be functional before I built the event handler. I just want to start getting things put on the master.